app/views/cards/_card_form.html.erb

```diff
@@ -47,11 +47,13 @@
           <%= f.text_area :content, class: 'form-control', rows: '5', placeholder: 'Fill in the blank prompt ...' %>
           <p class="help-block">Add blanks -> #blank here#'</p>
         </div>
-        <div class="form-group">
-          <%= f.label :status %>
-          <%= f.select(:status, Card.statuses.keys.map {|status| [status.titleize, status]}, {}, {:class => 'form-control'}) %>
-        </div>
-        <%= f.submit @card.card_type ? 'Update' : 'Create', class: 'btn btn-brand' %>
+        <% if @card.id %>
+          <div class="form-group">
+            <%= f.label :status %>
+            <%= f.select(:status, Card.statuses.keys.map {|status| [status.titleize, status]}, {}, {:class => 'form-control'}) %>
+          </div>
+        <% end %>
+        <%= f.submit @card.id ? 'Update' : 'Create', class: 'btn btn-brand' %>
```

Comment:
same: `@card.new_record?` is more expressive. There's also `.persisted?` if you don't want to swap the order of Update and Create